SIRF_data_preparation/data_QC.py

```diff
@@ -6,23 +6,31 @@
 
 Options:
   -h, --help
+  --srcdir=<path>        pathname (must have trailing slash!) [default: ./]
+  --skip_sino_profiles   do not plot the sinogram profiles
+  --transverse_slice=<i> idx [default: -1]
```

Comment:
@casperdcl `docopt` doesn't pick up this default (but says it's `None`). Any idea why?

Reply:
at least two spaces required

```suggestion
  --transverse_slice=<i>  idx [default: -1]
```